### PR TITLE
Write workflow run stats to CSV

### PR DESCRIPTION
**Pull request by NickVolynkin**
Part of #22

### CODE REVIEW

File: multivac/gather_job_data.py [hunk 1]
```diff
@@ -64,15 +78,51 @@ def gather_data(self):
...
+            json.dump(self.gathered_data, jsonfile, indent=2)
+
+    def write_csv(self):
         if not os.path.isdir(self.output_dir):
             os.makedirs(self.output_dir)
-        output_file = os.path.join(self.output_dir + '/gathered_data.json')
-        with open(output_file, 'w') as f:
-            json.dump(self.gathered_data, f, indent=2)
+        output_file = os.path.join(self.output_dir + '/workflows.csv')
+        fieldames = [
```
Comment (lastoCHka42) [inline multivac/gather_job_data.py:92]: typo: fieldames instead of field_n_ames

File: multivac/gather_job_data.py [hunk 2]
```diff
@@ -64,15 +78,51 @@ def gather_data(self):
...
+            'queued_at',
+            'started_at',
+            'completed_at',
+            'platform',
+            'runner_label',
+            'runner_name',
+            'runner_version',
+        ]
+        with open(output_file, 'w') as csvfile:
+            writer = csv.DictWriter(csvfile, fieldnames=fieldames)
```
Comment (lastoCHka42) [inline multivac/gather_job_data.py:105]: the same typo

File: multivac/gather_job_data.py [hunk 3]
```diff
@@ -1,51 +1,65 @@
...
+                        match = version_matcher.search(line)
+                        if match:
+                            runner_version = match.group(1)
+                            break
+            except FileNotFoundError:
+                print(f'no logs for job {job_id}')
 
             # Save data to dict
-            self.gathered_data[job['id']] = {
+            self.gathered_data[job_id] = {
```
Comment (lastoCHka42) [inline multivac/gather_job_data.py:61]: Good construction, could be used also in lines 72 and 76

**Review by lastoCHka42 — APPROVED** [review]

File: multivac/gather_job_data.py [hunk 4]
```diff
@@ -56,23 +70,59 @@ def gather_data(self):
...
+            for job_data in self.gathered_data.values():
+                writer.writerow(job_data)
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Gather data about GitHub workflows')
+    parser.add_argument(
+        '--write', choices=['json', 'csv'],
+        help='write gathered data in the specified format'
+    )
```
Comment (Totktonada) [inline multivac/gather_job_data.py:116]: Nit: It is --format in last_seen.py.

Comment (NickVolynkin) [reply multivac/gather_job_data.py]: Changed to `--format`

**Pull request merged**